driver-core/src/main/java/com/datastax/driver/core/CodecRegistry.java

```diff
@@ -294,7 +306,7 @@ public void onRemoval(RemovalNotification<CacheKey, TypeCodec<?>> notification)
      * Creates a new instance initialized with built-in codecs for all the base CQL types.
      */
     public CodecRegistry() {
-        this.codecs = new CopyOnWriteArrayList<TypeCodec<?>>(PRIMITIVE_CODECS);
+        this.codecs = new CopyOnWriteArrayList<TypeCodec<?>>();
```

Comment:
Putting this here because I can't comment on unmodified lines: the doc of the `codecs` field (line 297) is now outdated:
```java
/**
 * The list of registered codecs.
 * This list is initialized with the built-in codecs;
 * User-defined codecs are appended to the list.
 */
private final CopyOnWriteArrayList<TypeCodec<?>> codecs;
```